pkg/broker/broker.go

```diff
@@ -655,10 +657,44 @@ func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest,
 			return nil, err
 		}
 	}
-	// TODO: What data needs to be sent back on a response?
-	// Not clear what dashboardURL means in an AnsibleApp context
-	// operation should be the task id from the work_engine
-	return &ProvisionResponse{Operation: token}, nil
+
+	var response *ProvisionResponse
+
+	dashboardRedirectEnabled := func() bool {
+		var val interface{}
+		var dr bool
+
+		if len(spec.Alpha) == 0 {
+			return false
+		}
+		if val, ok = spec.Alpha["dashboard_redirect"]; !ok {
+			return false
+		}
+		if dr, ok = val.(bool); !ok {
+			return false
+		}
+		if a.brokerConfig.DashboardRedirector == "" {
+			warnMsg := fmt.Sprintf("Attempting to provision %v", spec.FQName) +
```

Comment:
I personally prefer log.Warningf and then you can remove the string concatenation.

Reply:
Do you have a preferred alternative to adding the strings? I really can't stand 130 char error messages, and that's all I care about :)

Reply:
You could do one of the following, use tick marks:

```golang
	warnMsg := `Attempting to provision %v, which has dashboard redirect enabled,
but no dashboard_redirector route was found in the broker's configmap.
Deploying without a dashboard_url.`

	log.Warningf(warnMsg, spec.FQName)
```
You could keep the concatenation but remove the sprintf using the Warningf version:

```golang
	warnMsg1 := "Attempting to provision %v, which has dashboard redirect enabled" +
		"but no dashboard_redirector route was found in the broker's " +
		"configmap. Deploying without a dashboard_url."

	log.Warningf(warnMsg1, "FQName")
```

Or you could reword the message to be shorter:

```golang

	// short message, no fqname
	log.Warning("Deploying without a dashboard_url, no dashboard_redirector route found")

	// short message with fqname
	log.Warningf("Deploying without a dashboard_url, no dashboard_redirector route found for %v", spec.FQName)
```